### PR TITLE
JDK 11 build

### PR DESCRIPTION
**Pull request by wmdietl**

### CODE REVIEW

**Comment by wmdietl** [comment]
@xingweitian Am I using the wrong docker image names?

**Comment by xingweitian** [comment]
> @xingweitian Am I using the wrong docker image names?

It should be `xingweitian/ubuntu-for-cfi-jdk11`, not `xingweitian/ubuntu-for-cfi-11`.

File: src/checkers/inference/InferenceMain.java [hunk 1]
```diff
@@ -298,7 +298,7 @@ private InferrableChecker getRealChecker() {
         if (realChecker == null) {
             try {
                 realChecker = (InferrableChecker) Class.forName(
-                        InferenceOptions.checker, true, ClassLoader.getSystemClassLoader()).newInstance();
```
Comment (zcai1) [inline src/checkers/inference/InferenceMain.java]: All changes on `InferenceMain.java `, `CnfVecIntSerializer.java`, and `JsonDeserializer.java` have already been merged into typetools, check https://github.com/typetools/checker-framework-inference/commit/65aa1cac6df1f9ab20cc75030292c9bde4936fce

File: .travis.yml [hunk 2]
```diff
@@ -17,20 +17,22 @@ env:
   global:
     - THIS_REPO=checker-framework-inference
     - OS=ubuntu
-    - JDKVER=jdk8
 
   matrix:
-    - GROUP=cfi-tests
-    - GROUP=downstream-ontology
-    - GROUP=downstream-units
+    - GROUP=cfi-tests JDKVER=jdk8
```
Comment (zcai1) [inline .travis.yml]: Reminder: we should update `.travis.yml` when pulling typetools/mater into opprop, by having groups with JDK8 and groups with JDK11.

File: .travis.yml [hunk 3]
```diff
@@ -17,20 +17,22 @@ env:
...
+    - GROUP=downstream-units JDKVER=jdk11
 
 # This is essential lest the script is ignored and Travis just runs gradle.
 install: true
 
 # The "docker run" command will pull if needed.
 # Running this first gives two tries in case of network lossage.
 before_script:
-- timeout 5m docker pull charleszhuochen/$OS-for-cfi-$JDKVER || true
+- timeout 5m docker pull xingweitian/$OS-for-cfi-$JDKVER || true
```
Comment (zcai1) [inline .travis.yml]: This is already in opprop/master

File: .travis.yml [hunk 4]
```diff
@@ -51,7 +53,7 @@ script:
 - echo "TRAVIS_PULL_REQUEST_SLUG=${TRAVIS_PULL_REQUEST_SLUG}"
 - echo "TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG}"
 - >-
-  docker run -v $HOME/.gradle:/root/.gradle -v $HOME/.m2:/root/.m2 charleszhuochen/$OS-for-cfi-$JDKVER /bin/bash -c "true &&
+  docker run -v $HOME/.gradle:/root/.gradle -v $HOME/.m2:/root/.m2 xingweitian/$OS-for-cfi-$JDKVER /bin/bash -c "true &&
```
Comment (zcai1) [inline .travis.yml]: This is already in opprop/master

**Comment by wmdietl** [comment]
@zcai1 Thanks for double checking!
For completeness, let me also add a reference to https://github.com/typetools/checker-framework-inference/pull/151

**Pull request closed without merging**